### PR TITLE
EVG-2041: move attach results out of s3put retry loop

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: command/s3_put.go [hunk 1]
```diff
@@ -234,10 +246,6 @@ retryLoop:
 					remoteName = fmt.Sprintf("%s%s", s3pc.RemoteFile, fname)
 				}
 
-				auth := &aws.Auth{
-					AccessKey: s3pc.AwsKey,
-					SecretKey: s3pc.AwsSecret,
-				}
 				s3URL := url.URL{
 					Scheme: "s3",
 					Host:   s3pc.Bucket,
```
Comment (bsamek) [inline command/s3_put.go]: unrelated to your CR, but 259 says "important to *not* wrap this error" and then wraps it

File: command/s3_put.go [hunk 2]
```diff
@@ -214,6 +223,9 @@ retryLoop:
 			return errors.New("s3 put operation canceled")
 		case <-timer.C:
 			filesList := []string{s3pc.LocalFile}
+			// reset to avoid duplicated uploaded references
+
+			uploadedFiles = []string{}
```
Comment (bsamek) [inline command/s3_put.go]: Move above 238, and delete space between it and its comment.

File: command/s3_put.go (hunk 1 above)
Comment (tychoish) [reply command/s3_put.go]: the not wrapping it was an artifact of the old retry mechanism, which is no longer relevant, so we've removed it.

File: command/s3_put.go (hunk 2 above)
Comment (tychoish) [reply command/s3_put.go]: aye aye

**Pull request merged**